lib/trace-agent.js

```diff
@@ -134,6 +133,9 @@ TraceAgent.prototype.runInSpan = function(name, labels, fn) {
  */
 TraceAgent.prototype.setTransactionName = function(name) {
   var rootSpan = cls.getRootContext();
+  if (rootSpan === SpanData.nullSpan) {
```

Comment:
It would be nice if we can remove the need to do this test. One way of doing this would be:
1. Make `SpanData#span` private.
2. Create a method on `SpanData` that transparently handles this.